### DISCUSSION TITLE
Dashboard Bug for deleted tweets

### DISCUSSION THREAD

**Discussion by sahiljaiswal02** [supabase/supabase#16827] [Contribute to Supabase]
# Bug report
On the Dashboard page the account of many people are not getting redirected to twitter. Example: @jim_bisenius

Add any other 
![1](https://github.com/supabase/supabase/assets/92286518/29bb70e7-fb89-4a68-9f40-a3d126a8478b)
![2](https://github.com/supabase/supabase/assets/92286518/8e282f72-6759-4172-a413-b4f0793e053f)
context about the problem here.

**Comment by AbhinawRatan** [comment]
I am working on this bug can anyone from the developer's team explain if there's any scarper used to collect all the tweets or if the tweet.json is a static JSON file?

**Comment by saltcod** [comment]
Yep its a static file @AbhinawRatan

**Reply by AbhinawRatan** [reply]
if thats the case then i have made a pull request for this issue

**Comment by sahiljaiswal02** [comment]
Can anyone suggest me in which file there might be this thing on which i can work on?

**Reply by AbhinawRatan** [reply]
I think the issue is solved by now
